react/features/deep-linking/components/DeepLinkingMobilePage.js

```diff
@@ -5,28 +5,21 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 
 import { translate, translateToHTML } from '../../base/i18n';
-import { Platform } from '../../base/react';
-import { URI_PROTOCOL_PATTERN } from '../../base/util';
+import { HideNotificationBarStyle, Platform } from '../../base/react';
 import { DialInSummary } from '../../invite';
-import HideNotificationBarStyle from './HideNotificationBarStyle';
 
-declare var interfaceConfig: Object;
+import { _TNS } from '../constants';
+import { generateDeepLinkingURL } from '../functions';
 
-/**
- * The namespace of the CSS styles of UnsupportedMobileBrowser.
- *
- * @private
- * @type {string}
- */
-const _SNS = 'unsupported-mobile-browser';
+declare var interfaceConfig: Object;
 
 /**
- * The namespace of the i18n/translation keys of UnsupportedMobileBrowser.
+ * The namespace of the CSS styles of DeepLinkingMobilePage.
  *
  * @private
  * @type {string}
  */
-const _TNS = 'unsupportedBrowser';
+const _SNS = 'deep-linking-mobile';
```

Comment:
What do TNS and SNS stand for?

Reply:
Hmm. Not 100% sure but maybe:
TNS - translation namespace
SNS - style namespace